### ISSUE TITLE
Trading UI doesn't appear

### ISSUE THREAD

**Issue by ianeinser** [openware/peatio#1437]
Hi,
i got a situation here. I run peatio with 'bundle exec rails server -p 3000' and it is displayed
=> Booting Puma
=> Rails 4.2.10 application starting in production on http://localhost:3000
=> Run `rails server -h` for more startup options
=> Ctrl-C to shutdown server
Puma starting in single mode...
* Version 3.11.4 (ruby 2.5.0-p0), codename: Love Song
* Min threads: 5, max threads: 5
* Environment: production
* Listening on tcp://localhost:3000
Use Ctrl-C to stop

then I run the trading UI with ''bundle exec rails server -p 4000'
WARNING: Use strings for Figaro configuration. false was converted to "false".
=> Booting Puma
=> Rails 5.1.6 application starting in production 
=> Run `rails server -h` for more startup options
Puma starting in single mode...
* Version 3.11.3 (ruby 2.5.0-p0), codename: Love Song
* Min threads: 5, max threads: 5
* Environment: production
* Listening on tcp://127.0.0.1:4000
Use Ctrl-C to stop
Started GET "/" for 127.0.0.1 at 2018-07-17 22:56:24 +0700
  
ActionController::RoutingError (No route matches [GET] "/"):

The RoutingError appears when I tried to access localhost:4000 from the browser..

my default nginx file which is located in /etc/nginx/site-enabled/default is

server {
  listen           80;
  server_name      plutonext.io;
  proxy_set_header Host plutonext.io;

  location ~ ^/(?:trading|trading-ui-assets)\/ {
    proxy_pass http://127.0.0.1:4000;
  }

  location / {
    proxy_pass http://127.0.0.1:3000;
  }

  error_page    500 502 503 504   /50x.html;
  location = /50x.html {
	root html;
  }
}

Any help is appreciated..

**Comment by risenforward** [comment]
Please make sure to add the next line to /etc/hosts.
127.0.0.1  plutonext.io
And you don't need to indicate port on url.
The nginx will take you to the trading ui if you go http://localhost/trading/usdbtc.
Thanks.

**Comment by ianeinser** [comment]
Hi risenforward,

Thanks for the input. However, i still got the error message at the trading ui side

puma (3.11.3) lib/puma/configuration.rb:225:in `call'
puma (3.11.3) lib/puma/server.rb:624:in `handle_request'
puma (3.11.3) lib/puma/server.rb:438:in `process_client'
puma (3.11.3) lib/puma/server.rb:302:in `block in run'
puma (3.11.3) lib/puma/thread_pool.rb:120:in `block in spawn_thread'
Started GET "/" for 127.0.0.1 at 2018-07-18 07:15:10 +0700
  
ActionController::RoutingError (No route matches [GET] "/"):

it seems nginx cannot forward the request or some works needed to be done for the routes. What should i do now?

**Comment by ianeinser** [comment]
Hi anyone or admin,

the problem is solved. I think it is a bug in v1.8. I just set
export RAILS_ENV=production
to 
export RAILS_ENV=development
and the trading runs even what i got is an empty chart.

I notice there is a huge different when the env is set differently. I saw more reports when in development mode when I run following steps
bundle exec rake db:create
bundle exec rake db:migrate
bundle exec rake currencies:seed
bundle exec rake markets:seed

and i didn't run bundle exec rake assets:precompile.

I know I might encounter this problem again when I set the environment to production mode. I hope someone can figure out the solution.
